itest.rb

```diff
@@ -1,7 +1,10 @@
 class Unknown
   def initialize(text)
     # Initalize hash    
-    @words = Hash.new
+      @words = Hash.new # two space for indentation
```

Comment:
__[rubocop]__
* Use 2 (not 4) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Use hash literal `{}` instead of `Hash.new`. [:ref](https://github.com/bbatsov/ruby-style-guide#literal-array-hash)